python/ql/src/semmle/python/dataflow/new/internal/DataFlowPrivate.qll

```diff
@@ -905,6 +905,18 @@ predicate storeStep(Node nodeFrom, Content c, Node nodeTo) {
   posOverflowStoreStep(nodeFrom, c, nodeTo)
   or
   kwOverflowStoreStep(nodeFrom, c, nodeTo)
+  or
+  jsonLoadsStoreStep(nodeFrom, c, nodeTo)
```

Comment:
Like with additional taint steps, I think we should use the same a mechanism to allow defining these in each framework implementation file (a minor nitpick I think).

Reply:
I completely agree that it cannot be in the core library like this. I believe that is what flow-summaries are for and that is the reason I have brought them up in this context.